src/v2/Apps/Fair/FairApp.tsx

```diff
@@ -123,7 +123,7 @@ const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
 
           <RouteTabs position="relative">
             <RouteTab
-              to={`/${fair.href}`}
+              to={fair.href}
```

Comment:
Was this a recent regression? Or did something in our underlying `RouteTab` change? I'm curious if this issue will start appearing elsewhere.

Reply:
It was just that I had used an `href` in an update to this page for the articles link. Then noticed we were manually interpolating the slugs and just bulk replaced them.